lib/samson/secrets/vault_server.rb

```diff
@@ -73,13 +73,12 @@ def sync!(other)
 
       def create_client
         VaultClientWrapper.new(
-          DEFAULT_CLIENT_OPTIONS.merge(
-            address: address,
-            ssl_cert_store: cert_store,
-            ssl_verify: tls_verify,
-            token: token,
-            versioned_kv: versioned_kv?
-          )
+          **DEFAULT_CLIENT_OPTIONS,
+          address: address,
+          ssl_cert_store: cert_store,
+          ssl_verify: tls_verify,
+          token: token,
+          versioned_kv: versioned_kv?
```

Comment:
The `VaultClientWrapper` class accepts keyword arguments in its initializer. Update this call to match.

https://github.com/zendesk/samson/blob/a370be781f1b0753ea2a0939b6b688fe63eed749/lib/samson/secrets/vault_client_wrapper.rb#L10